packages/qdrant-loader/src/qdrant_loader/core/qdrant_manager.py

```
@@ -215,27 +215,22 @@ def create_collection(self) -> None:
-                    embeddings_cfg = getattr(llm_settings, "embeddings", None)
-                    vs = (
-                        getattr(embeddings_cfg, "vector_size", None)
-                        if embeddings_cfg is not None
-                        else None
-                    )
-                    if isinstance(vs, int):
-                        vector_size = int(vs)
+                vs = self.settings.llm_settings.embeddings.vector_size
+                if vs is not None:
+                    vector_size = int(vs)
             except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)